tds-dll-common/pom.xml

```diff
@@ -16,42 +16,81 @@
 	</properties>
 
 	<dependencies>
+		<!-- Start: Program management integration dependency -->
+		<dependency>
+			<groupId>org.opentestsystem.shared</groupId>
+			<artifactId>prog-mgmnt-client</artifactId>
+			<version>${progman-client.version}</version>
+		</dependency>
+
+		<dependency>
+			<groupId>org.opentestsystem.shared</groupId>
+			<artifactId>prog-mgmnt-client-null-impl</artifactId>
+			<version>${progman-client.version}</version>
+		</dependency>
+		<!-- End: Program management integration dependency -->
+
 		<dependency>
 			<groupId>org.opentestsystem.shared</groupId>
 			<artifactId>shared-db</artifactId>
 		</dependency>
+
     	<dependency>
 			<groupId>org.opentestsystem.shared</groupId>
 			<artifactId>shared-tr-api</artifactId>
 		</dependency>
+
 		<dependency>
 			<groupId>org.springframework</groupId>
 			<artifactId>spring-context-support</artifactId>
 			<version>${org.springframework-version}</version>
 		</dependency>
+
 		<dependency>
 			<groupId>org.springframework</groupId>
 			<artifactId>spring-jdbc</artifactId>
 			<version>${org.springframework-version}</version>
 		</dependency>
+
+		<dependency>
+			<groupId>org.springframework.data</groupId>
+			<artifactId>spring-data-redis</artifactId>
+			<version>${spring-data-redis.version}</version>
+		</dependency>
```

Comment:
Added a spring/redis dependency that has a transitive dependency on spring 3.2.8, which should mesh well with our 3.2.9/3.2.1 mixed legacy spring versions.